projects/typesystem/projects/builtin_types/src/builtin_types.rb

```diff
@@ -25,7 +25,7 @@ def build_and_register!(
         casters_module = Util.constant_value(builtin_type_module, :Casters)
         caster_classes = if casters_module
                            Util.constant_values(casters_module, extends: Value::Processor)
```

Comment:
I'll add a TODO to just fix this in Util.constant_values instead so it can be fixed universally